bugsnag-android-core/src/main/java/com/bugsnag/android/internal/JsonHelper.kt

```diff
@@ -53,6 +54,8 @@ class JsonHelper private constructor() {
         fun deserialize(file: File): MutableMap<String, Any> {
             try {
                 FileInputStream(file).use { stream -> return deserialize(stream) }
+            } catch (ex: FileNotFoundException) {
```

Comment:
I wanted it to show which filename had problems in the case of generic IO errors, but to specifically pass through file-not-found so that it can be reliably caught by a higher level caller (because in some cases it's not a fatal error if the file doesn't exist).